app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+
+  def new
+    
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
@ivanraybuglosa I think you already updated this action in previous commit:
https://github.com/framgia831/rails-elearning-ivan/commit/d02abb19a36d235532ee6ae9be242b4e05ec60a5#diff-d5241d488259f32ecbe2f636133e5ddaR7